lib/authentication.js

```diff
@@ -52,7 +52,7 @@ module.exports = {
                req.logIn(user, function(err) {
                   debug('Got login from ' + user.username);
 
-                  // Redirect user to original URL. Fallback to '/' if DNE.
+                  // Redirect user to original URL. Fallback to '/' if DONE.
```

Comment:
This was used as "Does Not Exist", not "DONE"

Reply:
ohhh